rhpam73-image-streams.yaml

```diff
@@ -1,4 +1,4 @@
-kind: List
+kind: ImageStreamList
```

Comment:
Is this type of List new in OCP (and if so, which version)? If this has been supported for a while, I'm okay with it, however we'd also need to do this in RHDM (not just RHPAM).

Reply:
Apparently it is there since openshift 3.6: https://github.com/openshift/origin/blob/release-3.6/pkg/image/apis/image/types.go#L188

I'll push the prs for rhdm, ips and ds today.

Reply:
Ah, okay.